crdppf/lib/content.py

```diff
@@ -61,7 +92,7 @@ def set_documents(request, topicid, doctype, docids, featureinfo, geofilter, top
                 documents.append({"officialtitle": doc['officialtitle'], "title": doc['title'], "remoteurl": doc['remoteurl']})
                 if doc['doctype'] != u'legalbase' and doc['documentid'] not in doclist:
                     appendices.append(add_appendix(topicid, 'A'+str(len(appendices)+1), unicode(doc['officialtitle']).encode('iso-8859-1'),
-                        unicode(doc['title']).encode('iso-8859-1'), unicode(
+                                                   unicode(doc['title']).encode('iso-8859-1'), unicode(

```

Comment:
This is a huge indent...

Reply:
I agree, but flake says, that is, what's needed... 😀 ... so who am I to object... 😉